src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderManager.java

```diff
@@ -154,20 +158,20 @@ private List<CompositePoLine> getNonPackageLines(List<CompositePoLine> composite
     return compositePoLines.stream().filter(line -> !line.getIsPackage()).collect(toList());
   }
 
-  private Future<Void> finishProcessingEncumbrancesForOpenOrder(CompositePurchaseOrder compPO, CompositePurchaseOrder poFromStorage,
-                            RequestContext requestContext) {
+  private Future<Void> finishProcessingEncumbrancesForOpenOrder(CompositePurchaseOrder compPO,
+      CompositePurchaseOrder poFromStorage, RequestContext requestContext) {
     EncumbranceWorkflowStrategy strategy = encumbranceWorkflowStrategyFactory.getStrategy(OrderWorkflowType.PENDING_TO_OPEN);
-    Promise<Void> promise = Promise.promise();
-    strategy.processEncumbrances(compPO, poFromStorage, requestContext)
-      .onSuccess(result -> promise.complete())
-      .onFailure(t -> {
-        logger.error(t);
+    return strategy.processEncumbrances(compPO, poFromStorage, requestContext)
+      .onSuccess(v -> logger.info("Finished processing encumbrances to open the order"))
+      .recover(t -> {
+        logger.error("Error when processing encumbrances to open the order", t);
```

Comment:
let's add order identifier to all changed logs to simplify troubleshooting

Reply:
OK